doc/project/project_roles.rst

```diff
@@ -314,15 +314,37 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals:
+
+  * A collaborator from the same subsystem.
+  * Alternately another maintainer of another subsystem
+  * Approval by the assignee
+
+* Four eye principle on the organisation level.
+
+  * Changes or additions to common and shared code shall have reviews from
+    different organisations (at least one review and approval from a different
+    submitters' organisation).
+  * Changes or additions to hardware support (driver, SoC, boards) shall at
+    least have the merger be from a different organisation.
+  * Release engineers shall not merge code changes originating from the same
```

Comment:
> " Release engineers shall not merge code changes originating from the same"

Does this mean a) "Release engineers shall not merge code changes originating from *their* same organization"? or b) if both submitter and reviewer are from the same organization?

(in case of b) this would be a severe problem for quite a few areas where there just isn't so many involved developers. In some areas even getting two actual reviews is just plain difficult )

Reply:
it means that we shall not have submitter, approvers and mergers from the same organisation. It is recognised that we have some areas (beside hardware support and drivers, etc.) where this might be a problem with reviews, but in this case the only change introduced is that the merger is from a different org.